README.md

```diff
@@ -313,7 +313,7 @@ PYBIND11_MODULE(newtonraphsonpy, m) {
 
 Compile with
 
-```{.shell #build-py}
+```{.awk #build-py}
```

Comment:
not sure why this needs to be awk, further up ([244](https://github.com/NLESC-JCER/cpp2wasm/blame/master/README.md#L244)) there is another instance of .shell, but that's not used so maybe thats why it passes

Reply:
entangled does not know about [shell](https://github.com/entangled/entangled/blob/master/src/Config.hs#L37-L59), so I used awk as an alternative. The other `shell` commands are not entangled so can still use the `shell` naming. Made consistent in https://github.com/NLESC-JCER/cpp2wasm/pull/17/commits/129577d4eadb5e7fa6a8adb03295a7332104fe27